site/src/atomics/flex.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Flex Atomics
+description: Flex related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Flex Atomics
+
+Flex atomics are useful when composing layouts and patterns containing other components.
+
+| cssproperty       | value                                                                               | screensize |
+| ----------------- | ----------------------------------------------------------------------------------- | ---------- |
+| `flex-direction`  | `row`, `column`, `row-reverse`, `column-reverse`                                    | `tablet`   |
+| `justify-content` | `flex-start`, `flex-end`, `center`, `space-around`, `space-between`, `space-evenly` | `tablet`   |
+| `align-items`     | `flex-start`, `flex-end`, `center`, `baseline`, `stretch`                           | `tablet`   |
+| `align-self`      | `flex-start`, `flex-end`, `center`, `baseline`, `stretch`                           | `tablet`   |
+| `flex-wrap`       | `nowrap`, `wrap`, `wrap-reverse`                                                    | `tablet`   |
+| `align-content`   | `flex-start`, `flex-end`, `space-around`, `space-between`, `stretch`                | `tablet`   |
+| `flex-grow`       | `0`, `1`                                                                            | `tablet`   |
+| `flex-shrink`     | `0`, `1`                                                                            | `tablet`   |
+
+## Usage
+
+The following example uses a combination of mobile-first flex atomics to achieve a responsive card layout.
+
+```html
+<article
+	id="example-layout"
+	class="background-color-body-medium border border-radius-lg display-flex flex-direction-row"
+>
+	<div
+		class="background-color-info color-info-invert padding-md-tablet border-radius-lg display-flex align-items-center flex-shrink-0 flex-grow-0 "
+	>
+		<h1 class="font-size-h1">Hesiod</h1>
```

Comment:
What do you think about reducing the size a bit here? So it looks better on mobile

Reply:
I'm playing with the size right now. I'm thinking it can be h3, then h1 at tablet.

Reply:
Shouldn't be an h1 on this page.

Reply:
Does it really that matter? When I'm thinking about the cards in that layout it is _usually_ an icon on the left side, which doesn't take much space. So I think h2 would work fine for all screen sizes :)

Reply:
It matters in that any given page should only have a single `h1`. Which is not required but highly encouraged by the W3C: https://www.w3.org/TR/2014/REC-html5-20141028/sections.html#headings-and-sections

Reply:
Updated to h2

Reply:
> It matters in that any given page should only have a single `h1`. Which is not required but highly encouraged by the W3C: https://www.w3.org/TR/2014/REC-html5-20141028/sections.html#headings-and-sections

@wibjorn My comment was not questioning the "single `h1`element  on a page", I was referring to the idea of using different sizes on different screens 😄